docs/_links.rst

```diff
@@ -0,0 +1,25 @@
+.. _Astropy: https://www.astropy.org
+.. _BibTeX format: https://www.bibtex.com/g/bibtex-format
+.. _BibTeX: http://www.bibtex.org
+.. _conda: https://docs.conda.io
+.. _git: https://git-scm.com
+.. _Hinode: https://www.nasa.gov/mission_pages/hinode/index.html
+.. _Jupyter: https://jupyter.org
+.. _matplotlib: https://matplotlib.org
+.. _NumPy: https://numpy.org
+.. _pip: https://pip.pypa.io
+.. _pre-commit: https://pre-commit.com
+.. _PyPI: https://pypi.org
+.. _pytest: https://docs.pytest.org
+.. _Python: https://www.python.org
+.. _Read the Docs: https://readthedocs.org
+.. _SciPy: https://scipy.org
+.. _SolarSoft IDL: https://www.lmsal.com/solarsoft
+.. _Sphinx: https://www.sphinx-doc.org
+.. _SunPy: https://docs.sunpy.org/en/stable
+.. _tox: https://tox.wiki/en/latest
+.. _X-Ray Telescope: https://xrt.cfa.harvard.edu
+.. _XRT: https://xrt.cfa.harvard.edu/
+.. _XRTpy's documentation: https://xrtpy.readthedocs.io/en/latest
+.. _XRTpy's GitHub repository: https://github.com/HinodeXRT/xrtpy
+.. _Zenodo: https://zenodo.org
```

Comment:
This file pre-defines a bunch of links that we can use in the online documentation, like...
```
XRTpy is a Python_ package for the `X-Ray Telescope`_ on Hinode_.
```